plasmapy/formulary/parameters.py

```diff
@@ -61,12 +62,12 @@
 from plasmapy.utils.exceptions import PhysicsWarning
 
 
-def _grab_charge(ion, z_mean=None):
+def _grab_charge(ion: Particle, z_mean=None):
     """Utility function to merge two possible inputs for particle charge.
 
     Parameters
     ----------
-    ion : str or `plasmapy.particles.Particle`
+    ion : `plasmapy.particles.Particle`
```

Comment:
Ok, I have a couple more questions (sorry... I'm pretty new to using sphinx and docstrings). 
1) Is it ok to use the fully qualified name here (`plasmapy.particles.Particle`) instead of just `Particle`?
2) Should this be `` `plasmapy.particles.Particle` `` (with the backticks) or `~plasmapy.particles.Particle` (without the backticks, but with a preceeding tilde)? 

And what does that tilde even do? Is that part of the sphinx toolchain that makes clickable documentation links?

Reply:
It's better to use the fully qualified name, since not doing so could create linking issues when building the documentation.  The `~` indicates to `Sphinx` to just display the last element in the path.  So `` `~plasmapy.particles.Particle` `` will still link the fully qualified path but the link in the documentation will look like `Particle` instead of `plasmapy.particles.Particle`.

When to use `~` and not?  Well, it depends on the context of the current spot in the documentation.  If `Particle` makes sense within the current context then use it, otherwise use `plasmapy.particles.Particle`.  It's kind like using UCLA vs. University of California, Los Angeles.  When in context UCLA is fine.

Reply:
Thanks!